tests/python/openassetio_mediacreation/trait/test_entity.py

```diff
@@ -10,12 +10,12 @@
 
 import pytest
 
-from openassetio import Specification
+from openassetio import TraitsData
 
 ##
-## LocatableContentTrait
+## Traits
 ##
-
+from openassetio_mediacreation.trait import ClipTrait
```

Comment:
[consider] `ClipTrait` looks unused in this file? (and so comment above can be reverted too)

Reply:
Aye good shout [aab3343](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/14/commits/aab334339e0ae2c7967c2e5cc76d89b2afbad50c)